src/interchange/src/avro.rs

```diff
@@ -2083,7 +2083,7 @@ pub mod cdc_v2 {
     #[derive(Debug)]
     pub struct Decoder;
 
-    /// Builds the JSON for the row schema, which can be independenly useful.
+    /// Builds the JSON for the row schema, which can be independently useful.
```

Comment:
:)

Reply:
haha - it caught my eye while I was in there!